xwiki-platform-core/xwiki-platform-office/xwiki-platform-office-importer/src/main/resources/document-formats.js

```diff
@@ -19,196 +19,666 @@
-    "mediaType": "application/x-shockwave-flash",
-    "inputFamily": "DRAWING",
+    "name": "OpenDocument Text Template",
+    "extensions": [
+      "ott"
+    ],
+    "mediaType": "application/vnd.oasis.opendocument.text-template",
+    "inputFamily": "TEXT",
     "storeProperties": {
-      "DRAWING": {"FilterName": "draw_flash_Export"},
-      "PRESENTATION": {"FilterName": "impress_flash_Export"}
+      "TEXT": {
+        "FilterName": "writer8_template"
+      },
+      "WEB": {
+        "FilterName": "writerweb8_writer_template"
+      }
     }
   },
+  // New one
   {
-    "name": "HTML",
-    "extensions": ["html"],
-    "mediaType": "text/html",
+    "name": "OpenDocument Text Flat XML",
+    "extensions": [
+      "fodt"
+    ],
+    "mediaType": "application/vnd.oasis.opendocument.text-flat-xml",
     "inputFamily": "TEXT",
     "storeProperties": {
-      "SPREADSHEET": {"FilterName": "HTML (StarCalc)"},
-      "PRESENTATION": {
-        "FilterName": "impress_html_Export",
-        "FilterData": {"PublishMode": 0}
-      },
-      "TEXT": {"FilterName": "HTML (StarWriter)"}
+      "TEXT": {
+        "FilterName": "OpenDocument Text Flat XML"
+      }
     }
   },
   {
-    "name": "OpenDocument Text",
-    "extensions": ["odt"],
-    "mediaType": "application/vnd.oasis.opendocument.text",
+    "name": "Microsoft Word 2007-2013 XML",
+    "extensions": [
+      "docx"
+    ],
+    "mediaType": "application/vnd.openxmlformats-officedocument.wordprocessingml.document",
     "inputFamily": "TEXT",
-    "storeProperties": {"TEXT": {"FilterName": "writerweb8_writer"}}
+    "storeProperties": {
+      "TEXT": {
+        "FilterName": "MS Word 2007 XML"
+      }
+    }
   },
+  // New one
   {
-    "name": "OpenOffice.org 1.0 Text Document",
-    "extensions": ["sxw"],
-    "mediaType": "application/vnd.sun.xml.writer",
+    "name": "Word Open XML Document Template",
+    "extensions": [
+      "dotx"
+    ],
+    "mediaType": "application/vnd.openxmlformats-officedocument.wordprocessingml.template",
     "inputFamily": "TEXT",
-    "storeProperties": {"TEXT": {"FilterName": "StarOffice XML (Writer)"}}
+    "storeProperties": {
+      "TEXT": {
+        "FilterName": "MS Word 2007 XML"
+      }
+    }
   },
   {
-    "name": "Microsoft Word",
-    "extensions": ["doc"],
+    "name": "Microsoft Word 97-2003",
+    "extensions": [
+      "doc"
+    ],
     "mediaType": "application/msword",
     "inputFamily": "TEXT",
-    "storeProperties": {"TEXT": {"FilterName": "MS Word 97"}}
+    "storeProperties": {
+      "TEXT": {
+        "FilterName": "MS Word 97"
+      }
+    }
   },
   {
-    "name": "Microsoft Word 2007 XML",
-    "extensions": ["docx"],
-    "mediaType": "application/vnd.openxmlformats-officedocument.wordprocessingml.document",
-    "inputFamily": "TEXT"
+    "name": "HTML",
+    "extensions": [
+      "html"
+    ],
+    "mediaType": "text/html",
+    "inputFamily": "TEXT", // Standard file is using "WEB"
```

Comment:
I'm really not sure if we want to change this or not: here I kept our old value, but jodconverter file is using "WEB".

Reply:
I don't recall us changing this explicitly. The default value was probably TEXT before and now it's WEB. So I'd keep the default value.

Reply:
Ok I will change it.